LICENSE

```diff
@@ -0,0 +1,5 @@
+(c) Copyright 2017 by Radomir Dopieralski.
```

Comment:
I'm not sure it should be copyrighted all by me, when you are adding all those files here. Maybe you should add yourself as well?

Reply:
It was only because I'm always afraid of repositories without licensing at all.
It's not important if you protect it with simple copyright, Apache or MIT license or whatever.
There are too many bad people around these days.